src/overlay.repos

```diff
@@ -8,4 +8,7 @@ repositories:
         url: https://github.com/tud-cor-sr/ros2-vtem_control
     mocap_optitrack:
         type: git
-        url: https://github.com/tud-cor-sr/ros2-mocap_optitrack.git
\ No newline at end of file
+        url: https://github.com/tud-cor-sr/ros2-mocap_optitrack.git
+    elastica:
```

Comment:
I think elastica should only be cloned if the argument is set to true. So please move the elastica repo to its own `elastica.repos` file, any basically copy lines 27-36 in the Dockerfile for the `elastica.repos` and only run it when the argument is set to true

Reply:
Yeah, thanks for pointing out. I forgot to set cloning elastica depending on the arg value. I'll fix that in the next commit.

Reply:
I did that in the latest commit